### PR TITLE
Add parameters for loads in table modification

### PR DESCRIPTION
**Pull request by lilian-houdelet**

### CODE REVIEW

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js [hunk 1]
```diff
@@ -72,6 +72,16 @@ const TabularModificationDialog = ({
 
     const { reset } = formMethods;
 
+    const toModificationFromEnum = useCallback(
```
Comment (etiennehomer) [inline src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js:75]: You should synchronize with David for the method name and the behavior

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js [hunk 2]
```diff
@@ -107,6 +117,11 @@ const TabularModificationDialog = ({
                     const value = row[key];
                     if (key === 'equipmentId') {
                         modification[key] = value;
+                    } else if (key === 'loadType') {
+                        modification[key] = toModificationFromEnum(
```
Comment (etiennehomer) [inline src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js:121]: you can use toModificationOperation here with "get enum id" method

File: src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js [hunk 3]
```diff
@@ -72,6 +72,16 @@ const TabularModificationDialog = ({
 
     const { reset } = formMethods;
 
+    const getValueFromEnum = useCallback(
```
Comment (etiennehomer) [inline src/components/dialogs/network-modifications/tabular-modification/tabular-modification-dialog.js:75]: You can use the same method as https://github.com/gridsuite/gridstudy-app/pull/1773 : 
const getEnumIdFromLabel = useCallback(
        (label, enumValues) =>
            enumValues.find(
                (es) => intl.formatMessage({ id: es.label }) === label
            )?.id,
        [intl]
    );

File: src/translations/fr.json [hunk 4]
```diff
@@ -699,8 +701,10 @@
     "ImportModifications": "Importer des modifications",
     "TabularModificationError": "Erreur lors de la modification tabulaire",
     "TabularModificationGeneratorSkeletonComment" : "#,,,,true | false",
+    "TabularModificationLoadSkeletonComment": "#,Auxiliaire | Fictif,,",
     "equipmentId": "ID",
-    "activePower": "p (MW)",
+    "ActivePower": "p (MW)",
+    "activePower": "P constant (MW)",
```
Comment (etiennehomer) [inline src/translations/fr.json:707]: You can remove it nope ?

File: src/translations/fr.json [hunk 5]
```diff
@@ -131,9 +131,11 @@
     "ActivePowerT3WSide1": "p1 (MW)",
     "ActivePowerT3WSide2": "p2 (MW)",
     "ActivePowerT3WSide3": "p3 (MW)",
+    "ActivePowerSide3": "p3 (MW)",
     "ReactivePower": "q (MVar)",
-    "ConstantReactivePower": "Q constant (MVar)",
-    "ConstantActivePower": "P constant (MW)",
+    "reactivePower": "Q constant (MVar)",
```
Comment (etiennehomer) [inline src/translations/fr.json:136]: to be removed ?

File: src/translations/fr.json [hunk 6]
```diff
@@ -699,8 +701,10 @@
     "ImportModifications": "Importer des modifications",
     "TabularModificationError": "Erreur lors de la modification tabulaire",
     "TabularModificationGeneratorSkeletonComment" : "#,,,,true | false",
+    "TabularModificationLoadSkeletonComment": "#,Auxiliaire | Fictif,,",
     "equipmentId": "ID",
-    "activePower": "p (MW)",
+    "ActivePower": "p (MW)",
```
Comment (etiennehomer) [inline src/translations/fr.json:706]: with the changes in network-modification-server, you can keep activePower

File: src/services/study/network-modifications.js [hunk 7]
```diff
@@ -485,8 +485,8 @@ export function modifyLoad(
             equipmentId: id,
             equipmentName: toModificationOperation(name),
             loadType: toModificationOperation(loadType),
-            activePower: toModificationOperation(activePower),
-            reactivePower: toModificationOperation(reactivePower),
+            constantActivePower: toModificationOperation(activePower),
```
Comment (etiennehomer) [inline src/services/study/network-modifications.js:488]: constantActivePower: toModificationOperation(constantActivePower), ?

**Review by etiennehomer — APPROVED** [review]

**Review by ayolab — APPROVED** [review]

**Pull request merged**